# Experiment Setup

title: "slice_apple"
description: ""
regions:
  - name: "lab"
    min_x: -3.0
    max_x: 6.0
    min_y: -10.0
    max_y: 3.0
    min_z: 0.5
    max_z: 2
objects:
  - name: "apple"
    priors:
      - name: "lab"
        weight: 1.0
  - name: "knife"
    priors:
      - name: "lab"
        weight: 1.0
frames:
  - "grasp_knife"
  - "slice_apple"
action_history:
  - "idle"
